podman-image-daily/Containerfile

```diff
@@ -57,6 +57,10 @@ RUN systemctl disable coreos-platform-chrony-config.service && \
     # Append the chrony conf dir location to the default config
     echo "confdir /etc/chrony.d" >> /etc/chrony.conf
 
+# Disable the systemd resolver, the unit must be masked other fedora enables it again
+# https://github.com/containers/podman-machine-os/issues/18
+RUN systemctl mask systemd-resolved.service
```

Comment:
worth adding a test to ensure it stays disabled?

Reply:
yes, but I first still need to figure out what the actual "recommend" way is to disable that thing...